ajax/libs/Swiper/package.json

```diff
@@ -16,9 +16,9 @@
   "npmName": "swiper",
   "npmFileMap": [
     {
```

Comment:
not entirely sure but I think we should keep the `basePath`. 
```suggestion
    {
      "basePath": "",
```

Reply:
Emm.... althought it has a little weird, I have commited it please check again~